.azure/azure-pipelines.ci.yml

```diff
@@ -673,6 +673,15 @@ stages:
       platform: windows
       tls: schannel
       allocFail: 100
+  - template: ./templates/run-spinquic.yml
+    parameters:
+      image: windows-2022
+      platform: windows
+      tls: schannel
+      allocFail: 100
```

Comment:
does this use the same `CXPLAT_ALLOC` routines, or have its own allocator?

Reply:
Same ones. LoLa uses a different "datapath" but same "platform" as winuser.